python/nano/src/bigdl/nano/utils/inference/pytorch/model_utils.py

```diff
@@ -82,3 +84,37 @@ def export_to_onnx(model, input_sample=None, onnx_path="model.onnx", dynamic_axe
                                 }
     default_onnx_export_args.update(kwargs)
     torch.onnx.export(model, input_sample, onnx_path, **default_onnx_export_args)
+
+
+class FP32InferenceModelWrapped(pl.LightningModule):
```

Comment:
Shall we change the `inference_method_name` method (e.g., "predict" or "inference") to call `forward`?

Reply:
We don't need to do this in `FP32InferenceModelWrapped` since it is totally an internal class (i.e. users won't know this class, there won't be any external API return this class or any class inherited from this class.)

To support `model_int8.inference_method_name()`, I will try to change some code in  `AcceleratedLightningModule`.